webui-page/webui.js

```diff
@@ -411,6 +427,26 @@ function setPosSlider(position, duration) {
   pos.innerHTML = format_time(slider.value);
 }
 
+function handleMediaPositionStart(e) {
+  const offset = touchRangeOffset(e);
+  setPosSlider(offset, document.getElementById("mediaPosition").max);
+};
+
+function handleMediaPositionMove(e){
+  const offset = touchRangeOffset(e);
+  setPosSlider(offset, document.getElementById("mediaPosition").max);
+};
+
+function handleMediaPositionEnd(e) {
+  const offset = touchRangeOffset(e);
+  setPosSlider(offset, document.getElementById("mediaPosition").max);
+  (offset != false) && send("set_position", offset);
```

Comment:
hi, in `touchRangeOffset(e)` there is a return false, it is to prevent error just incase some lag happens on someone's computer and it doesn't remove the listener in time. 
It was there is my previous update functions too.
I will remove it since you tested and it is now useless.